db/fdb_fend.c

```diff
@@ -3208,7 +3208,7 @@ static int fdb_cursor_find_sql_common(BtCursor *pCur, Mem *key, int nfields,
         }
 
         if (pCur->ixnum == -1) {
```

Comment:
@dorinhogea: This is one of two places that needed to change to get the "write_remsql_names" test to pass on this branch.  Please review for correctness.

Reply:
The change is correct. When I cherry-picked the onepass performance patch from 3.34.0 onto 7.0, I had to make the exact same change. See https://github.com/bloomberg/comdb2/pull/2413.